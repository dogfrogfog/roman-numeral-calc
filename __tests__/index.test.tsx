import { render } from '@testing-library/react'

import Home from '@/pages/index'

describe('Component: Home page', () => {
  it('renders component correctly', () => {
    const { container } = render(<Home />)

    expect(container).toMatchSnapshot()
  })
})
