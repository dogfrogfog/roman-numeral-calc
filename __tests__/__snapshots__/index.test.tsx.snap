// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Home page renders component correctly 1`] = `
<div>
  <div
    class="container"
  >
    <main
      class="mainSection"
    >
      <div
        class="section"
      >
        <h3
          class="title"
        >
          Numeric input
        </h3>
        <input
          class="input"
          max="1000"
          min="1"
          type="number"
          value=""
        />
      </div>
      <div
        class="section"
      >
        <h3
          class="title"
        >
          Roman output
        </h3>
        <p
          class="outputValue"
        />
      </div>
    </main>
  </div>
</div>
`;
